docs/conf.py

```diff
@@ -29,5 +29,7 @@
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = "pydata_sphinx_theme"
-html_title = "astro-provider-ray"
+html_theme = "furo"
```

Comment:
Just for my understanding any reason why we are changing theme

Reply:
It looks nicer than the old theme.

Also, wanted the table of contents as a sidebar instead of tabs on the top